### PR TITLE
edid-decode: new, 0.1+git20230804

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

edid-decode: new, 0.1+git20230804

https://git.linuxtv.org/edid-decode.git/about/

Package(s) Affected
-------------------

- edid-decode

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-utils/edid-decode/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=edid-decode
+PKGSEC=utils
+PKGDEP=""
```
Comment (MingcongBai) [inline app-utils/edid-decode/autobuild/defines]: Not possible?

File: app-utils/edid-decode/spec [hunk 2]
```diff
@@ -0,0 +1,3 @@
+VER=0.1+git20230804
+SRCS="git::commit=5f723267e04deb3aa9610483514a02bcee10d9c2::https://git.linuxtv.org/edid-decode.git"
+CHKSUMS="sha256::c10823c940641404cd6eb9c801d43b307e87e9252e8eded66eed8be750c4f884"
```
Comment (MingcongBai) [inline app-utils/edid-decode/spec:3]: Missing CHKUPDATE - use Anitya.

File: app-utils/edid-decode/spec [hunk 3]
```diff
@@ -0,0 +1,3 @@
+VER=0.1+git20230804
```
Comment (MingcongBai) [inline app-utils/edid-decode/spec]: Why the snapshot? Why not 0.1?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/edid-decode/spec (hunk 3 above)
Comment (jiegec) [reply app-utils/edid-decode/spec]: It has no release tag.

File: app-utils/edid-decode/autobuild/defines (hunk 1 above)
Comment (jiegec) [reply app-utils/edid-decode/autobuild/defines]: It's a simple program with minimal dependencies.

File: app-utils/edid-decode/spec (hunk 2 above)
Comment (jiegec) [reply app-utils/edid-decode/spec:3]: The package has no release tags, and does not appear on https://release-monitoring.org/

File: app-utils/edid-decode/autobuild/defines (hunk 1 above)
Comment (MingcongBai) [reply app-utils/edid-decode/autobuild/defines]: Then it would still have at least `glibc`.

File: app-utils/edid-decode/spec (hunk 3 above)
Comment (MingcongBai) [reply app-utils/edid-decode/spec]: In that case, the portion before +git should be 0.

File: app-utils/edid-decode/spec (hunk 2 above)
Comment (MingcongBai) [reply app-utils/edid-decode/spec:3]: Got it, please leave a `# FIXME:` comment.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**